src/ralph/lib/table/table.py

```diff
@@ -24,10 +24,10 @@ class Table(object):
     Generating contents for table based on predefined columns and queryset.
 
     Example:
-        >>> table = Table(queryset, ['id', 'name'])
+        >>> table = Table(queryset, ['id', ('name', 'My field name')])
```

Comment:
That description is fine, but I'd also add here a remark: "see __init__'s docstring for additional info" (or something like that).